content/soarian/r4/base/entities/organization.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Organization | R4 API
+---
+
+# Organization
+
+* TOC
+{:toc}
+
+## Overview
+
+The Organization resource describes a healthcare entity.
+
+Soarian Clinicals<sup>®</sup> supports a read-only Application Programming Interface (API). This API accepts `GET` and `POST` based [search] and `GET` based [read] interactions. The response represents the most current information about the patient that is charted in Soarian Clinicals<sup>®</sup> at the time of the query.
+
+The search results include the following fields if they contain values:
+
+* [Id](https://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Identifier](https://hl7.org/fhir/r4/organization-definitions.html#Organization.identifier){:target="_blank"}
+* [Active](https://hl7.org/fhir/r4/organization-definitions.html#Organization.active){:target="_blank"}
+* [Type](https://hl7.org/fhir/r4/organization-definitions.html#Organization.type){:target="_blank"}
+* [Name](https://hl7.org/fhir/r4/organization-definitions.html#Organization.name){:target="_blank"}
+* [Contact Information](https://hl7.org/fhir/r4/organization-definitions.html#Organization.telecom){:target="_blank"}
+* [Address](https://hl7.org/fhir/r4/organization-definitions.html#Organization.address){:target="_blank"}
+  
+
+
+
+## Search
+
+Search for Organization resources that meet the specified query parameters:
+
+    GET /Organization?:parameters
+
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true)%>
+
+### Parameters
+
+ Name          | Required?                                    | Type            | Description
+---------------|----------------------------------------------|-----------------|----------------------------------------------------------------------------------------------------------------------------
+ `_id`         | This name, or `address`                      | [`token`]       | The logical resource ID associated with the resource.
+ `name`        | This name, or `address`				      | [`string`]      | The start of the name of the organization. 
+ `address`     | This name, or `address`				      | [`string`]      | The start of either the address line, city, state, postal code, or country of the organization. 
+ 
+
+
+ 
+
+
+
+### Headers
+
+<%= headers fhir_json: true %>
+
+### Example Search by ID
+
+#### Request
+
+    GET https://fhir-myrecord-sc.sandboxcerner.com/r4/3f2aca24-87f3-4eac-a6d7-1f75247e6b43/Organization?_id=HCU.1
```

Comment:
Cross checking with Patient resource, we should ideally provide URLs which are public sandbox and are reachable. Patient resource has something like
https://fhir-myrecord-sc.cerner.com/r4/2f8f5ec1-b7b8-4be5-ae27-e308284dd9c1/Patient/A879904FD2FE4B2D90C89FDA84E1285F'

Update in all other instances likewise